airbyte-integrations/connectors/source-mysql-v2/src/main/kotlin/io/airbyte/integrations/source/mysql/MysqlJdbcSelectQuerier.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.source.mysql
+
+import io.airbyte.cdk.read.JdbcSelectQuerier
+import io.airbyte.cdk.read.SelectQuerier
+import io.airbyte.cdk.read.SelectQuery
+import io.github.oshai.kotlinlogging.KotlinLogging
+import io.micronaut.context.annotation.Primary
+import javax.inject.Singleton
+
+private val log = KotlinLogging.logger {}
+
+@Singleton
+@Primary
+class MysqlJdbcSelectQuerier(val base: JdbcSelectQuerier) : SelectQuerier by base {
+
+    override fun executeQuery(
+        q: SelectQuery,
+        parameters: SelectQuerier.Parameters
+    ): SelectQuerier.Result {
+        log.info { "Executing query: ${q.sql}, bindings: ${q.bindings}" }
+        return base.executeQuery(q, SelectQuerier.Parameters(fetchSize = Int.MIN_VALUE))
+    }
+}
```

Comment:
Didn't we delete this recently?

Reply:
I think there's some merge issue - i'll remove it here too